### PR TITLE
Fix DeprecationWarning

### PR DESCRIPTION
**Pull request by CrafterKolyan**
## Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contribution guide](https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md)?
- [ ] Did you check the code style? `catalyst-make-codestyle && catalyst-check-codestyle` (`pip install -U catalyst-codestyle`).
- [x] Did you make sure to update the docs? We use Google format for all the methods and classes.
- [ ] Did you check the docs with `make check-docs`?
- [x] Did you write any new necessary tests?
- [ ] Did you check that your code passes the unit tests `pytest .` ? 
- [ ] Did you add your new functionality to the docs?
- [ ] Did you update the [CHANGELOG](https://github.com/catalyst-team/catalyst/blob/master/CHANGELOG.md)?

## Description

image

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves an existing feature)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

PS
- [x] I know, that I could [join slack](https://join.slack.com/t/catalyst-team-core/shared_invite/zt-d9miirnn-z86oKDzFMKlMG4fgFdZafw) for pull request discussion.

### CODE REVIEW

File: catalyst/callbacks/optimizer.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def grad_step(
         for group, wd in zip(optimizer.param_groups, optimizer_wds):
             if wd > 0:
                 for param in group["params"]:
-                    param.data = param.data.add(-wd * group["lr"], param.data)
+                    param.data = param.data.add(param.data, alpha=-wd * group["lr"])
```
Comment (CrafterKolyan) [inline catalyst/callbacks/optimizer.py]:
```suggestion
                    param.data = param.data.add(param.data,
                                                alpha=-wd * group["lr"])
```

File: catalyst/callbacks/optimizer.py [hunk 2]
```diff
@@ -137,7 +137,8 @@ def grad_step(
         for group, wd in zip(optimizer.param_groups, optimizer_wds):
             if wd > 0:
                 for param in group["params"]:
-                    param.data = param.data.add(-wd * group["lr"], param.data)
+                    param.data = param.data.add(param.data,
+                                                alpha=-wd * group["lr"])
```
Comment (CrafterKolyan) [inline catalyst/callbacks/optimizer.py]:
```suggestion
                    param.data = param.data.add(
                        param.data, alpha=-wd * group["lr"]
                    )
```

File: catalyst/callbacks/optimizer.py [hunk 3]
```diff
@@ -137,7 +137,7 @@ def grad_step(
         for group, wd in zip(optimizer.param_groups, optimizer_wds):
             if wd > 0:
                 for param in group["params"]:
-                    param.data = param.data.add(-wd * group["lr"], param.data)
+                    param.data.add_(param.data, alpha=-wd * group["lr"])
```
Comment (CrafterKolyan) [inline catalyst/callbacks/optimizer.py]:
```suggestion
                    param.data.add_(other=param.data, alpha=-wd * group["lr"])
```

**Review by Scitator — DISMISSED** [review]
Hi, that's for the PR. could you please fix the tests?

**Comment by CrafterKolyan** [comment]
> Hi, that's for the PR. could you please fix the tests?

@Scitator tests have been fixed by themselves

**Pull request merged**